### PR TITLE
task card ui redesign

### PR DESCRIPTION
**Pull request by Pavangbhat**
# updated design
image

#114

### CODE REVIEW

File: src/components/tasks/card/card.module.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+.cardTitle {
+  font-size: 1.5rem;
+  font-weight: 500;
+  color: #041187;
+}
+
+.cardSpecialFont {
+  font-style: Medium;
+  font-size: 20px;
```
Comment (sumitd94) [inline src/components/tasks/card/card.module.scss:37]: why not `rem` here?

File: src/components/tasks/card/card.module.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  font-weight: 500;
+  color: #aeaeae;
+}
+
+.cardStatusFont {
+  color: #00a337;
+  font-size: 20px;
```
Comment (sumitd94) [inline src/components/tasks/card/card.module.scss:44]: `rem`?

File: src/components/tasks/card/card.module.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  font-weight: 501;
+}
+.contributorImage {
+  height: 45px;
+  width: 45px;
+  border-radius: 40%;
+}
+
+.cardStrongFont {
+  font-size: 20px;
```
Comment (sumitd94) [inline src/components/tasks/card/card.module.scss:54]: `rem`?

**Review by sumitd94 — CHANGES_REQUESTED** [review]
Few changes requested in CSS, the rest all look good. Good job 🥇

File: src/components/tasks/card/card.module.scss [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.cardItems {
+  span {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 2px 2px;
```
Comment (ankushdharkar) [inline src/components/tasks/card/card.module.scss:24]: Nit: `margin: 2px;`

File: src/components/tasks/card/card.module.scss [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  font-style: Medium;
+  font-size: 1.5rem;
+  font-weight: 500;
+  color: #aeaeae;
+}
+
+.cardStatusFont {
+  color: #00a337;
+  font-size: 1.5rem;
+  font-weight: 501;
```
Comment (ankushdharkar) [inline src/components/tasks/card/card.module.scss:45]: This seems new.

File: src/components/tasks/card/card.module.scss [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+  }
+}
+.cardTitle {
+  font-size: 1.5rem;
+  font-weight: 500;
+  color: #041187;
+}
+
+.cardSpecialFont {
+  font-style: Medium;
```
Comment (ankushdharkar) [inline src/components/tasks/card/card.module.scss:36]: Do we need this?

File: src/components/tasks/card/card.module.scss [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+}
+.contributorImage {
+  height: 45px;
+  width: 45px;
+  border-radius: 40%;
+}
+
+.cardStrongFont {
+  font-size: 1.5rem;
+  font-weight: 501;
```
Comment (ankushdharkar) [inline src/components/tasks/card/card.module.scss:55]: 501?

File: src/components/tasks/card/index.tsx [hunk 8]
```diff
@@ -29,42 +24,56 @@ const Card: FC<Props> = ({ content }) => {
 
   const localStartedOn = new Date(parseInt(startedOn, 10) * 1000);
   const fromNowStartedOn = moment(localStartedOn).fromNow();
-  const readableStartedOn = `${localStartedOn.toLocaleDateString()}, ${localStartedOn.toLocaleTimeString()}`;
 
   const localEndsOn = new Date(parseInt(endsOn, 10) * 1000);
   const fromNowEndsOn = moment(localEndsOn).fromNow();
-  const readableEndsOn = `${localEndsOn.toLocaleDateString()}, ${localEndsOn.toLocaleTimeString()}`;
+
+  const statusFontColor = status === 'active' || 'assigned' || 'completed' || 'pending' ? '#00a337' : '#f83535';
```
Comment (ankushdharkar) [inline src/components/tasks/card/index.tsx:31]: Can we use constants here please?

File: src/components/tasks/card/index.tsx [hunk 9]
```diff
@@ -29,42 +24,56 @@ const Card: FC<Props> = ({ content }) => {
...
-        <span className={classNames.onhover}>
-          {informationElement('Estimated completion', readableEndsOn)}
+      </div>
+      <div className={classNames.cardItems}>
+        <span>
+          <Image
+            src="/calendar-icon.png"
+            alt="calendar icon"
+            width="25px"
+            height="25px"
```
Comment (ankushdharkar) [inline src/components/tasks/card/index.tsx:50]: Nit: Can these go into a variable?

File: src/components/tasks/card/index.tsx [hunk 10]
```diff
@@ -29,42 +24,56 @@ const Card: FC<Props> = ({ content }) => {
...
-        <span className={classNames.onhover}>
-          {informationElement('Started', readableStartedOn)}
+        <span>
+          <span className={classNames.cardSpecialFont}>Assignee:</span>
+          <span className={classNames.cardStrongFont}>{assignee}</span>
+          <span>
+            <img
+              className={classNames.contributorImage}
+              src={assigneeProfilePic}
+              alt="No contributor"
```
Comment (ankushdharkar) [inline src/components/tasks/card/index.tsx:71]: Seems like the wrong alt text, especially if we have the contributor image show up

**Review by ankushdharkar — CHANGES_REQUESTED** [review]
Please check the comments

File: src/components/tasks/card/index.tsx (hunk 10 above)
Comment (Pavangbhat) [reply src/components/tasks/card/index.tsx:71]: shall I show up contributor name for alt?

File: src/components/tasks/card/index.tsx (hunk 9 above)
Comment (Pavangbhat) [reply src/components/tasks/card/index.tsx:50]: fixed
image

File: src/components/tasks/card/index.tsx (hunk 8 above)
Comment (Pavangbhat) [reply src/components/tasks/card/index.tsx:31]: done 
image

File: src/components/tasks/card/card.module.scss (hunk 7 above)
Comment (Pavangbhat) [reply src/components/tasks/card/card.module.scss:55]: my bad fixed

File: src/components/tasks/card/card.module.scss (hunk 6 above)
Comment (Pavangbhat) [reply src/components/tasks/card/card.module.scss:36]: no my bad

File: src/components/tasks/card/card.module.scss (hunk 5 above)
Comment (Pavangbhat) [reply src/components/tasks/card/card.module.scss:45]: fixed

File: src/components/tasks/card/card.module.scss (hunk 4 above)
Comment (Pavangbhat) [reply src/components/tasks/card/card.module.scss:24]: fixed

File: src/components/tasks/card/index.tsx (hunk 8 above)
Comment (sumitd94) [reply src/components/tasks/card/index.tsx]: its better if we name the constant all uppercase (good practise)
```javascript
const ACTIVE = 'active'
```

Comment (sumitd94) [reply src/components/tasks/card/index.tsx]: Also can we please move the constants in a separate file?

Comment (Pavangbhat) [reply src/components/tasks/card/index.tsx]: That will done. Thank you for letting me know the best practices 😃

**Review by ankushdharkar — APPROVED** [review]
Looks great! Thanks 🎉

**Pull request merged**